implementations/rust/ockam/ockam_api/src/cloud/project/projects.rs

```diff
@@ -0,0 +1,62 @@
+use crate::cloud::project::models::OrchestratorVersionInfo;
+use crate::cloud::project::Project;
+use ockam_core::async_trait;
+use ockam_node::Context;
+
+#[async_trait]
+pub trait Projects {
```

Comment:
Perhaps this name could be named `ProjectsOrchestratorApi`.

Reply:
Yeah, I just moved it as is, but renaming is a good suggestion

Reply:
We have a lot of similar names that we need to change